src/libraries/System.Security.Cryptography/ref/System.Security.Cryptography.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
-    <NoWarn>$(NoWarn);SYSLIB0026</NoWarn>
+    <NoWarn>$(NoWarn);SYSLIB0026;SYSLIB5006</NoWarn>
```

Comment:
I am surprised you needed to do this. If everything that consumes `MLDsa` is also experimental in the `ref`, then it should propagate.

Does this indicate we have an API in the `ref` that is not experimental that should be?

Reply:
That's probably what it was saying, but since I fixed it a while ago, it looks like I can remove this.  Good catch.